pom.xml

```diff
@@ -226,7 +226,7 @@
       <versionx.org.jboss.arquillian.container.arquillian-weld-embedded>4.0.0.Final</versionx.org.jboss.arquillian.container.arquillian-weld-embedded>
       <versionx.org.jboss.jboss-common-core>2.5.0.Final</versionx.org.jboss.jboss-common-core>
       <versionx.org.jboss.jboss-transaction-spi>8.0.0.Final</versionx.org.jboss.jboss-transaction-spi>
-      <versionx.org.jboss.logging.jboss-logging-processor>2.2.1.Final</versionx.org.jboss.logging.jboss-logging-processor>
+      <versionx.org.jboss.logging.jboss-logging-processor>3.0.1.Final</versionx.org.jboss.logging.jboss-logging-processor>
```

Comment:
We should just remove the `versionx` property, I don't think they serve a purpose anymore \cc @pruivo

Reply:
Is it possible to remove one of the properties at least so we only have the version number defined in a single place?

Reply:
yes, sure